pkg/library/flatten/network/devfile.go

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) 2019-2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package network
+
+import (
+	"fmt"
+
+	dw "github.com/devfile/api/pkg/apis/workspaces/v1alpha2"
+	devfilev2 "github.com/devfile/api/pkg/devfile"
+)
+
+const (
+	SupportedDevfileSchemaVersion = "2.0.0" // TODO what should this actually be at this point?
```

Comment:
```
// TODO what should this actually be at this point?
```

I'd like to know the answer on that as well, I've just been using `2.0.0` on the dashboard side :laughing:

Reply:
Realistically, it should be something like `2\..*\..*`, assuming there's no backwards-incompatible changes.

Reply:
Replaced this with a basic `^2\..*` regexp.